parts/k8s/windowsazurecnifunc.tests.ps1

```diff
@@ -0,0 +1,17 @@
+. $PSScriptRoot\windowsazurecnifunc.ps1
+
+Describe 'GetBroadestRangesForEachAddress' {
+
+    It "Values '<Values>' should return '<Expected>'" -TestCases @(
+        @{ Values = @('10.240.0.0/12', '10.0.0.0/8'); Expected = @('10.0.0.0/8', '10.240.0.0/12')}

```

Comment:
aha - I thought that would get reordered. unit test proves it. This should be fine though

Reply:
Yes, didn't think that would be a big deal.
I made sure the re-ordering and filtering is reflected in the CSE logs as well

i pulled this from a deployment i just did
2019-10-08T21:49:45.9841253+00:00: Downloaded file to c:\k\azurecni\azure-vnet.zip
Filtering CNI config exception list values to work aroudn WS2019 issue processing rules. Original exception list: 10.240.0.0/12 10.240.0.0/12 10.0.0.0/8, processed exception list: 10.0.0.0/8 10.240.0.0/12
2019-10-08T21:49:50.3516533+00:00: Write kubelet startfile with pod CIDR of